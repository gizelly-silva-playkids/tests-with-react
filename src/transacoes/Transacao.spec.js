import React from 'react';
import { render } from '@testing-library/react';

import Transacao from './Transacao';

describe('TransacaoComponent ', () => {
   it('should be display snapshot\'s component', () => {
      const { container } = render(<Transacao 
         data="03/03/2022"
         tipo="saque"
         valor="400"
      />)

      expect(container.firstChild).toMatchSnapshot()
   });
})