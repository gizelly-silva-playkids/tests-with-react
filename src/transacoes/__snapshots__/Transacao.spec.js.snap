// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TransacaoComponent   1`] = `
<div
  class="transacao-container"
>
  <p>
    03/03/2022
  </p>
  <p>
    saque
  </p>
  <p>
    R$ 
    400
  </p>
</div>
`;

exports[`TransacaoComponent  should be display snapshot's component 1`] = `
<div
  class="transacao-container"
>
  <p>
    03/03/2022
  </p>
  <p>
    saque
  </p>
  <p>
    R$ 
    400
  </p>
</div>
`;
